corehq/form_processor/interfaces/dbaccessors.py

```diff
@@ -157,6 +157,9 @@ def soft_delete_forms(self, form_ids, deletion_date=None, deletion_id=None):
     def soft_undelete_forms(self, form_ids):
         return self.db_accessor.soft_undelete_forms(self.domain, form_ids)
 
+    def write_blob_bucket(self, attachment):
```

Comment:
originally did it for consistency, but suppose we could totally remove this in a week https://github.com/dimagi/commcare-hq/pull/13639/commits/058e2df0938f72b32a52f4853b7cda56af3557d7